packages/in_app_purchase/lib/src/billing_client_wrappers/billing_client_wrapper.dart

```diff
@@ -171,25 +176,31 @@ class BillingClient {
   /// [`BillingFlowParams`](https://developer.android.com/reference/com/android/billingclient/api/BillingFlowParams)
   /// instance by [setting the given
   /// skuDetails](https://developer.android.com/reference/com/android/billingclient/api/BillingFlowParams.Builder.html#setskudetails)
-  /// and [the given
-  /// accountId](https://developer.android.com/reference/com/android/billingclient/api/BillingFlowParams.Builder.html#setAccountId(java.lang.String)).
+  /// , [the given
```

Comment:
Nit: this is hard to read; let's put the comma on the previous line.

Reply:
done